cli/command/image/build.go

```diff
@@ -255,7 +321,14 @@ func runBuild(dockerCli *command.DockerCli, options buildOptions) error {
 
 	var body io.Reader = progress.NewProgressReader(buildCtx, progressOutput, 0, "", "Sending build context to Docker daemon")
 
-	authConfigs, _ := dockerCli.GetAllCredentials()
```

Comment:
This removes the only call of `GetAllCredentials()` so we should delete it from `DockerCli`.